### PR TITLE
sponsor list to represent how much they contribute to the team, added sponsors and removed sponsors, changed charles sponsor to represent whole family, contact me messages broncobots email not MCA's email, twoColumnGallery added (might replace gallery later to do any configuration) , robots list game from that year, stars on robots page represent number of awards won, attempted to correct 2021Logo in assets page, replaced 2016 robot image with highres version

### PR DESCRIPTION
**Pull request by Mallen220**
sponsor list to represent how much they contribute to the team, added sponsors and removed sponsors, changed charles sponsor to represent whole family, contact me messages broncobots email not MCA's email, twoColumnGallery added (might replace gallery later to do any configuration) , robots list game from that year, stars on robots page represent number of awards won, attempted to correct 2021Logo in assets page, replaced 2016 robot image with highres version

### CODE REVIEW

File: .github/workflows/jekyll.yml [hunk 1]
```diff
@@ -25,8 +25,8 @@ jobs:
       with:
         token: ${{ secrets.GITHUB_TOKEN }}
         jekyll_src: './'
-        build_dir: _site
         build_only: true
+        build_dir: _site
```
Comment (j0n5m1th) [inline .github/workflows/jekyll.yml:29]: This shouldn't need to change. Please keep it as it was.

File: .github/FUNDING.yml [hunk 2]
```diff
@@ -0,0 +1,12 @@
+## These are supported funding model platforms
```
Comment (j0n5m1th) [inline .github/FUNDING.yml:1]: Don't merge this file until its configured correctly - we don't need a placeholder.

File: _data/awards.yml [hunk 3]
```diff
@@ -0,0 +1,84 @@
+- Year: 2022
```
Comment (j0n5m1th) [inline _data/awards.yml:1]: Since there are other off season awards listed below, then please include CTTD 2022.

File: _data/resources.yml [hunk 4]
```diff
@@ -1,26 +1,52 @@
 - name: The Blue Alliance
```
Comment (j0n5m1th) [inline _data/resources.yml:1]: Could add another mechanical design resource: https://www.reca.lc/

File: _data/sponsors.yml [hunk 5]
```diff
@@ -23,100 +23,133 @@
 
 
 - tier_name: Company Sponsors
-  size: small 
+  size: small
   description: |-
-    These are the companys are sponsor us.
+    These are the company's are sponsor us.
+
+    (In order of amount donated)
```
Comment (j0n5m1th) [inline _data/sponsors.yml:30]: We don't need to call out order of amount donated.

File: _data/sponsors.yml [hunk 6]
```diff
@@ -23,100 +23,133 @@
 
 
 - tier_name: Company Sponsors
-  size: small 
+  size: small
   description: |-
-    These are the companys are sponsor us.
+    These are the company's are sponsor us.
```
Comment (j0n5m1th) [inline _data/sponsors.yml:28]: check spelling / typos, both here and throughout

long term, perhaps we can get grammar check integrated with github actions

File: _data/sponsors.yml [hunk 7]
```diff
@@ -23,100 +23,133 @@
...
-      profile: https://soundworksrec.com/index.html
-    - name: United States of America Navy
-      image: ../images/sponsors/Navy.jpg
-      description: In America's Navy, water and salt flow through our veins in the same proportion as the sea. That mighty force is the lifeblood of the greatest Navy ever to sail upon it, slip beneath it or fly above it. It shapes Americans from every corner of the nation into Sailors with the courage to protect liberty back home. Bonding us with the commitment to serve with honor. The sea propels every one of our men and women to defeat or defend. To comfort or to awe. To victory or to the rescue.
-    - name: Veterans of Foreign Wars
-      image: ../images/sponsors/VFW.png
-      description: The Veterans of Foreign Wars, formally the Veterans of Foreign Wars of the United States, is an organization of US war veterans, who, as military service members fought in wars, campaigns, and expeditions on foreign land, waters, or airspace.
-      profile: https://www.vfw.org/
+
+#    - name: GitHub
```
Comment (j0n5m1th) [inline _data/sponsors.yml:114]: Please remove the commented out sponsors.

File: _data/sponsors.yml [hunk 8]
```diff
@@ -23,100 +23,133 @@
...
-    - name: Linda Pickett
-      description: Personal Sponsor
     - name: Team 1987 Mentors
       description: Personal Sponsors
-    - name: Ted Cahall
-      description: Personal Sponsor
-    - name: Tracy Bornman PC
-      description: Personal Sponsor
+
+#    - name: Citizens of Lee's Summit
```
Comment (j0n5m1th) [inline _data/sponsors.yml:152]: Please remove commented out sponsors.

File: _includes/footer.html [hunk 9]
```diff
@@ -38,36 +38,39 @@
...
-  color: black;   
+  background-color: white;
+  color: black;
   border: 2px solid #830506;
 }
 </style>
 <footer class="footer">
     <div class="container">
+
+<!--TODO FIND BETTER ALTERNATIVE TO FORMSUBMIT.CO-->
```
Comment (j0n5m1th) [inline _includes/footer.html:49]: Please remove this commented section and create a new GitHub issue to replace this with something that works better.

File: _robots/2022Robot.md [hunk 10]
```diff
@@ -14,7 +14,7 @@ features:
       icon: fa-robot
     - label: ~95% high goal shooting accuracy
       icon: fa-crosshairs
-rating: 2
+rating: 4
```
Comment (j0n5m1th) [inline _robots/2022Robot.md:17]: Can we remove the `<!--price ... -->` comment from these files?
What does the `rating` attribute mean in these files?

Should the bulleted lists below be unordered lists? ex.
```
<h2>Robot capabilities</h2>
<ul>
  <li>Four wheel...</li>
  ...
</ul>
```

**Comment by j0n5m1th** [comment]
@Mallen220 
For future reference, PRs should only have related changes.
For example, this PR has many unrelated changes and could be broken up into multiple PRs:
  - Robots & awards updates
  - Sponsors updates
  - Resources updates

This will help reviewers get through PRs quicker & focus on details.  A lot can get missed when reviewing large PRs.  Git commit history will also be more clear.

File: _data/sponsors.yml (hunk 6 above)
Comment (j0n5m1th) [reply _data/sponsors.yml:28]: This should be `companies`.
I also think this is a redundant description.  It is the sponsors page & the `tier_name` describes this.  It should be evident that these companies that sponsor the team.  Consider removing this description.

File: SponsorUs.md [hunk 11]
```diff
@@ -41,7 +41,7 @@ show_sidebar: false
 </head>
 <body>
     <div class="row">
-        <div class="column" style="background-color:#fafafa;">
+        <div class="column" style="background-color:white;">
```
Comment (Mallen220) [inline SponsorUs.md:44]: this is an example. Ignore me.

**Review by kenschenke — APPROVED** [review]

**Pull request merged**